x/mongo/driver/topology/pool_test.go

```diff
@@ -226,20 +226,23 @@ func TestPool(t *testing.T) {
 						return
 					default:
 						loopCtx, loopCancel := context.WithTimeout(getCtx, 3*time.Second)
-						_, _ = p.get(loopCtx)
+						c, err := p.get(loopCtx)
```

Comment:
Would it also help if we increased the number of connections created by `bootstrapConnections`?

Reply:
I can increase the number, but it shouldn't change anything, there should be enough connections available now.